Storage Provider Incubation Center/README.md

```diff
@@ -47,11 +47,11 @@
 
  ### Is it suitable for you? 
 
- <img width="1288" alt="Rules and Requirements" src="https://user-images.githubusercontent.com/75723122/125607988-7553fec8-83c4-4bc8-bcce-8f19c57aefc0.png">
+Rules and Requirements.png![Rules and Requirements](https://user-images.githubusercontent.com/75723122/127090179-75fee051-b36f-475d-9029-4c996889b2f0.png)
 
  #### Storage Provider Incubation Center Program (Track 1)
 
- The storage provider incubation center do welcome new or small storage providers to join immediately, you need to seal at least 64 sectors per day (2TiB）to join this program, when a storage provider's storage power reached 1PiB or has participated this program over 90days, then the storage providers has to withdraw from this program, and the others who meet the requirements can be joined.
+ The storage provider incubation center do welcome new or small storage providers to join immediately, you need to seal at least 64 sectors per day (2TiB）to join this program, when a storage provider's storage power reached 1PiB or run to the end of the certain phase, then the storage providers has to withdraw from this program, and the others who meet the requirements can be joined.
 
```

Comment:
The storage provider incubation center is designed to nurture and grow new and small storage providers by providing technical and personalized support to grow your business. To join the program, you will need to seal 64 sectors per day (2TiB) and provide your own hardware infrastructure. A storage provider can participate the program for a total of 90days or until a 1PiB storage power has been reached, whatever comes first.

Reply:
The deadline for submitting applications has been extended to August 31, so the end date of one phase is now taken as the end date of continuous participation of storage providers (or 1PiB has been reached). So that each stage can be clearly distinguished（ Instead of everyone running for 90 days)